src/js/helpers/LoadHelpers.js

```diff
@@ -239,7 +238,7 @@ export function getParams(path, manifest) {
     }
 
     let params = {
-        'originalLanguagePath': ORIGINAL_LANGUAGE_PATH
+        'originalLanguagePath': ''
```

Comment:
The params variable is being removed and ORIGINAL_LANGUAGE_PATH was undefined anyway.